### PR TITLE
[trivial] lnrpc: reorder num_inactive_channels

### PR DESCRIPTION
**Pull request by halseth**
Small fixup from https://github.com/lightningnetwork/lnd/pull/1782

Move it close to num_active_channels for an output from lncli getinfo
that is easier to parse.

### CODE REVIEW

File: lnrpc/rpc.proto [hunk 1]
```diff
@@ -1063,6 +1063,9 @@ message GetInfoResponse {
     /// Number of active channels
     uint32 num_active_channels = 4 [json_name = "num_active_channels"];
 
+    /// Number of inactive channels
+    uint32 num_inactive_channels = 15 [json_name = "num_inactive_channels"];
```
Comment (cfromknecht) [inline lnrpc/rpc.proto]: IMO it's more important to keep the proto definitions in increasing order, so it stays clear what the last allocate number is. This also helps avoid gaps (where did 7 go?) and other inconsistencies.

In the past, we've reordered fields by defining a display struct at the RPC level with the desired order and mapping the fields into that before printing. There are a number of fields in getinfo that I think would benefit from a reordering, and this provides a better long-term solution by decoupling the display from the proto definitions

Comment (halseth) [reply lnrpc/rpc.proto]: Agreed! 👍

**Comment by halseth** [comment]
Closing per https://github.com/lightningnetwork/lnd/pull/2164#discussion_r232560960

**Comment by Roasbeef** [comment]
Alternatively, we can just re-order it on the command line, similar to how we handle doing custom responses in order to print hex rather than base58: https://github.com/lightningnetwork/lnd/blob/master/cmd/lncli/commands.go#L1985-L1993

**Pull request closed without merging**